consumers/hidden-test-consumer/src/hidden_test_consumer/process.py

```
@@ -237,29 +247,55 @@
             yield ProcessRequest(status=Status.ERROR, message="Error bundling tests")
             raise e
 
+        try:
+            # create record in the database
+            url = f"{self.backend_url}/api/internal/v1/problems/{self.problem_id}/hidden-test-bundle/"
+            headers = {"X-API-KEY": self.api_key, "Content-Type": "application/json"}
+            data: HiddenTestBundleData = {
+                "s3_path": f"processed/{self.problem_id}/hidden-tests.zip",
+                "test_count": len(self.input_seqs),
+            }
+            response = requests.post(url, headers=headers, json=data)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)